pulsar-broker/src/main/java/org/apache/pulsar/broker/namespace/NamespaceService.java

```diff
@@ -1137,6 +1137,10 @@ public CompletableFuture<Boolean> isServiceUnitOwnedAsync(ServiceUnitId suName)
                 new IllegalArgumentException("Invalid class of NamespaceBundle: " + suName.getClass().getName()));
     }
 
+    /**
+     * @Deprecated This method is only used by test. call "isServiceUnitActiveAsync" is better.
+     */
+    @Deprecated
     public boolean isServiceUnitActive(TopicName topicName) {
         try {
```

Comment:
would it make sense to use something like `isServiceUnitActiveAsync(topicName)..get(conf.getMetadataStoreOperationTimeoutSeconds(), SECONDS);`

Reply:
@lhotari 

I think it is a good suggestion. Already edit this method to make the test that calls the method works better.

Could you take a look again?